web/api/v1/api.go

```diff
@@ -1373,6 +1373,17 @@ func (api *API) rules(r *http.Request) apiFuncResult {
 	returnAlerts := typ == "" || typ == "alert"
 	returnRecording := typ == "" || typ == "record"
 
+	excludeAlertsParam := strings.ToLower(r.URL.Query().Get("exclude_active_alerts"))
+
+	if excludeAlertsParam == "" {
+		excludeAlertsParam = "false"
+	}
+
+	excludeAlerts, err := strconv.ParseBool(excludeAlertsParam)
+	if err != nil {
+		excludeAlerts = false
```

Comment:
Would it be better if we return 400 incase of parsing failure?

Reply:
+1

Reply:
Done